### PR TITLE
Add vcluster import command

### PR DESCRIPTION
**Pull request by neogopher**
Closes ENG-2088

### CODE REVIEW

File: cmd/vclusterctl/cmd/import.go [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+		Long:  description,
+		Args:  loftctlUtil.VClusterNameOnlyValidator,
+		RunE: func(cobraCmd *cobra.Command, args []string) error {
+			return cmd.Run(cobraCmd.Context(), args)
+		},
+	}
+
+	importCmd.Flags().StringVar(&cmd.VClusterClusterName, "cluster", "", "Cluster name of the cluster the virtual cluster is running on")
+	importCmd.Flags().StringVar(&cmd.VClusterNamespace, "namespace", "", "The namespace of the vcluster")
+	importCmd.Flags().StringVar(&cmd.Project, "project", "", "The project to import the vcluster into")
```
Comment (FabianKramm) [inline cmd/vclusterctl/cmd/import.go:48]: Can we find this out automatically? E.g. if there is only 1 project we use that

File: cmd/vclusterctl/cmd/import.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+		Use:   "import" + loftctlUtil.VClusterNameOnlyUseLine,
+		Short: "Imports a vcluster into a vCluster.Pro project",
+		Long:  description,
+		Args:  loftctlUtil.VClusterNameOnlyValidator,
+		RunE: func(cobraCmd *cobra.Command, args []string) error {
+			return cmd.Run(cobraCmd.Context(), args)
+		},
+	}
+
+	importCmd.Flags().StringVar(&cmd.VClusterClusterName, "cluster", "", "Cluster name of the cluster the virtual cluster is running on")
```
[the remark this answers is not in the document]
Comment (neogopher) [reply cmd/vclusterctl/cmd/import.go:69]: Done.

File: cmd/vclusterctl/cmd/import.go [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+	importCmd.Flags().StringVar(&cmd.VClusterNamespace, "namespace", "", "The namespace of the vcluster")
```
[the remark this answers is not in the document]
Comment (neogopher) [reply cmd/vclusterctl/cmd/import.go:70]: That's already the current behaviour.

File: cmd/vclusterctl/cmd/import.go (hunk 1 above)
Comment (neogopher) [reply cmd/vclusterctl/cmd/import.go:71]: Done.

**Review by FabianKramm — APPROVED** [review]
@neogopher thanks for the PR! LGTM!

**Pull request merged**